ale_linters/cpp/clangcheck.vim

```diff
@@ -20,7 +20,7 @@ function! ale_linters#cpp#clangcheck#GetCommand(buffer) abort
     " being generated. These are only added if no build directory can be
     " detected.
     return '%e -analyze %s'
-    \   . (empty(l:build_dir) ? ' -extra-arg -Xclang -extra-arg -analyzer-output=text' : '')
+    \   . (empty(l:build_dir) ? ' --extra-arg="-Xclang" --extra-arg="-analyzer-output=text" --extra-arg="-fno-color-diagnostics"': '')
```

Comment:
I reckon it's worth removing the double quotes here, as double quotes can sometimes break commands on Windows. Processing double quotes in a shell in Windows is the responsibility of the command instead of the shell, which causes issues where you sometimes end up passing `"foo"` to a Windows program instead of `foo`.

Reply:
Double quotes removed.